cmake/set_amrex_hydro_options.cmake

```diff
@@ -0,0 +1,17 @@
+# Set amrex hydro options
+
+set(HYDRO_SPACEDIM 3)
```

Comment:
Reuse `AMReX_SPACEDIM` here to be consistent, especially with external AMReX (e.g., spack builds)